pipelines/trigger/main.py

```diff
@@ -191,25 +186,34 @@ def get_env() -> dict:
     }
 
 
-def get_args(args: List[str] = None) -> argparse.Namespace:
-    """Get args from command line args
-    Args:
-        event (dict): Event payload.
-        context (google.cloud.functions.Context): Metadata for the event.
+def sandbox_run(args: List[str] = None) -> aiplatform.PipelineJob:
+    """Trigger a Vertex Pipeline run from a (local) compiled pipeline definition.
+    Returns the PipelineJob object of the triggered pipeline run.
+    Usage: python main.py --template_path=pipeline.json --enable_caching=true
     """
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--payload", help="Path to the payload JSON file", type=str)
-    return parser.parse_args(args)
-
-
-def sandbox_run() -> aiplatform.PipelineJob:
     logging.basicConfig(level=logging.DEBUG)
 
-    args = get_args()
-
-    # Load JSON payload into a dictionary
-    with open(args.payload, "r") as f:
-        payload = json.load(f)
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--template_path", help="Path to the payload JSON file", type=str
+    )
+    parser.add_argument("--enable_caching", type=str, default=None)
+
+    # Get commandline args
+    args = parser.parse_args(args)
+
+    # If empty value for enable_caching provided on commandline default to None
+    if args.enable_caching == "":
+        args.enable_caching = None
```

Comment:
Throw warning

Reply:
Why? Value can be `True` or `False` or `None`, not just True/False